### DISCUSSION TITLE
task 1 lab 3.01

### DISCUSSION THREAD

**Discussion by rbecerrav** [rramosp/ai4eng.v1#29] [03 - GENERACION DE MODELOS]
Hola profe Raul en este ejercicio es hacerlo en una linea.
yo lo hice como se muestra a continuacion,pero tengo el presentimiento que la intencion era no usar condicionales for o if 
pero no supe como crear un vector de booleanos con dos condiciones, solo era capaz con uno por ejemplo 
`X[:,0]<t[0]` me entrega un vector de booleanos, pero no se como agregar la otra condicion
el codigo me quedo asi
``` 
def predict(X, t):
   return np.array([0 if X[i,0]<t[0] and X[i,1]>t[1] else 1 for i in range(0,len(X))])
```

**Comment by rramosp** [comment]
hola rbecerrav, 

si quieres filtrar un array con varias condiciones puedes hacerlo de la siguiente manera

![image](https://user-images.githubusercontent.com/13835425/122996974-21658a80-d371-11eb-8054-89f809bb2fcd.png)

era esa tu duda??

**Reply by rbecerrav** [reply]
si profesor ya vi donde tenia mi error de sintaxis. muchas gracias
